http/handler.go

```diff
@@ -39,6 +41,9 @@ func Handler(core *vault.Core) http.Handler {
 	mux.Handle("/v1/sys/", handleLogical(core, true, nil))
 	mux.Handle("/v1/", handleLogical(core, false, nil))
 
+	// This is probably wrong in quite a few ways - unsure how to register it properly? Something like Mounting, maybe?
```

Comment:
Yeah, there is a configuration file option which I use in command/server.go. However, the configuration is not available in this scope. So, something else needs to be done, I'm just not sure what. I was thinking that maybe I could reuse the mount mechanics, not requiring that the user does it manually.